package.json

```diff
@@ -64,6 +60,8 @@
     "lodash.defaults": "3.1.1",
     "react": "0.13.1",
     "react-hammerjs": "0.2.2",
+    "react-imageloader": "1.2.0",
+    "react-loadermixin": "1.0.4",
```

Comment:
`ImageLoader` is used in `discover.jsx`. `LoaderMixin` is a dependency of the image loader.